tensorflow/lite/micro/kernels/depthwise_conv_test.cc

```diff
@@ -175,6 +147,35 @@ void TestDepthwiseConvQuantizedPerChannel(
                                               output_dims_count, conv_params,
                                               1.0, tensors_size, tensors));
 }
+#endif  // !defined(VISION_P6)
+
+#if !defined(XTENSA)  // Needed to avoid build errors from unsused functions.
```

Comment:
let's expand this comment:

```
// Xtensa kernels do not support float activations., and the corresponding tests are
// disabled. As a result, helper functions that are only needed for float kernel tests
// also need to be ifdef'd out to avoid build errors due to unused functions.
```

Reply:
got it !